### PR TITLE
[Hotfix] Blob Update counter-attacks

### PR DESCRIPTION
**Pull request by DeityLink**
Half were bugs we found when we forced a blob event earlier today.

The other half were ideas that came to me since incidentally I got to actually play blob for the first time.
- Blobs cannot destroy docking ports anymore.
- The Blob Overmind's HUD now properly indicates how much health his core has.
- Added a lot of messages to Blob Overminds that trigger under a variety of conditions. Such as blobs near their core dying (10 seconds cooldown), a special blob that they've placed having been destroyed, or their core getting damaged (3 seconds cooldown).
- If there are multiple Blob Overminds in the world, they get instantly notified by the death of one of them. They also get notified by the death of any blob core, whether it had an overmind or not.

### CODE REVIEW

File: code/game/gamemodes/blob/blobs/core.dm [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /obj/effect/blob/core
 	name = "blob core"
 	icon_state = "core"
+	desc = "Some big pulsating blob creature thingy"
```
Comment (gbasood) [inline code/game/gamemodes/blob/blobs/core.dm:4]: w e w

File: code/game/gamemodes/blob/blobs/node.dm [hunk 2]
```diff
@@ -1,9 +1,11 @@
 /obj/effect/blob/node
 	name = "blob node"
 	icon_state = "node"
+	desc = "Some pulsating blob creature thingy"
```
Comment (gbasood) [inline code/game/gamemodes/blob/blobs/node.dm:4]: w  e  w

File: code/game/gamemodes/blob/blobs/resource.dm [hunk 3]
```diff
@@ -1,6 +1,7 @@
 /obj/effect/blob/resource
 	name = "resource blob"
 	icon_state = "resource"
+	desc = "Some smoke-producing blob creature thingy"
```
Comment (gbasood) [inline code/game/gamemodes/blob/blobs/resource.dm:4]: jesus

**Comment by DeityLink** [comment]
I like my description naming theme.

**Pull request merged**